project/Dependencies.scala

```diff
@@ -5,14 +5,15 @@ object Dependencies {
 
   val Scala212 = "2.12.17"
   val Scala213 = "2.13.10"
-  val CrossScalaVersions = Seq(Dependencies.Scala213, Dependencies.Scala212)
+  val Scala3 = "3.1.3"
+  val CrossScalaVersions = Seq(Scala213, Scala212, Scala3)
```

Comment:
There is still one project that doesn't compile. akka-discovery-consul
Doesn't really look like scalac errors. Many of theses:
```
[warn] Caught: java.lang.AssertionError: assertion failed: failure to resolve inner class:
[warn] externalName = org.immutables.value.Value$Derived,
[warn] outerName = org.immutables.value.Value,
[warn] innerName = Derived
[warn] owner.fullName = org.immutables.value.Value
[warn] while parsing /Users/patrik/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/orbitz/consul/consul-client/1.5.3/consul-client-1.5.3.jar(com/orbitz/consul/option/QueryOptions.class) while parsing annotations in /Users/patrik/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/orbitz/consul/consul-client/1.5.3/consul-client-1.5.3.jar(com/orbitz/consul/option/QueryOptions.class)
[error] Bad symbolic reference. A signature
[error] refers to Value/T in package org.immutables.value which is not available.
[error] It may be completely missing from the current classpath, or the version on
[error] the classpath might be incompatible with the version used when compiling the signature.
```

Reply:
found it: https://github.com/akka/akka-management/pull/1112/commits/ee6f74ff7b6e37c38665fc499a5e2e8e803ce0c7